contrib/xstow/template.py

```diff
@@ -0,0 +1,13 @@
+pkgname = "xstow"
+pkgver = "1.1.0"
+pkgrel = 0
+build_style = "gnu_configure"
+make_dir = "."
+hostmakedepends = ["automake"]
+makedepends = ["ncurses-devel"]
+pkgdesc = "Extended replacement of GNU Stow"
```

Comment:
this description is not very descriptive, it should say what the software does, not what it's a replacement of